src/entities/BlockTune/types/BlockTuneSerialized.ts

```diff
@@ -0,0 +1,16 @@
+import { BlockTuneName } from './BlockTuneName';
+
+/**
+ * BlockTuneSerialized represents a serialized version of a BlockTune.
+ */
+export interface BlockTuneSerialized {
+  /**
+   * The name of the tune.
+   */
+  name: BlockTuneName;
```

Comment:
it should be a regular string after serialisation I guess?

Reply:
I agree with you. I'll fix it soon